static/src/javascripts/projects/commercial/modules/creatives/fabric-expandable-video-v2.js

```diff
@@ -0,0 +1,161 @@
+                );
+            }
+
+            if (isOpen) {
+                $('.ad-exp__close-button', adSlot).addClass('button-spin');
+                $('.ad-exp__open-chevron', adSlot).addClass('chevron-down');
+                $ad.css('height', openedHeight);
+                $fabricExpandableVideo.addClass('creative--open');
+                $('.slide-video, .slide-video .ad-exp__layer', adSlot)
+                    .css('height', openedHeight)
+                    .addClass('slide-video__expand');
+            } else {
+                $('.ad-exp__close-button', adSlot).removeClass('button-spin');
+                $('.ad-exp__open-chevron', adSlot).removeClass('chevron-down');
+                $ad.css('height', closedHeight);
+                $fabricExpandableVideo.removeClass('creative--open');
+                $('.slide-video, .slide-video .ad-exp__layer', adSlot)
+                    .css('height', closedHeight)
+                    .removeClass('slide-video__expand');
+            }
+
+            isClosed = !isOpen;
+
+            setTimeout(() => {
+                $('#YTPlayer').attr('src', videoSrcAutoplay);
+            }, 1000);
+        };
+
+        bean.on(adSlot, 'click', '.ad-exp__open', () => {
+            fastdom.write(() => {
+                open(isClosed);
+            });
+        });
+
+        bean.on(
+            adSlot,
+            'click',
+            '.video-container__cta, .creative__cta',
+            () => {
+                fastdom.write(() => {
+                    open(false);
+                });
+            }
+        );
+
+        return fastdom.write(() => {
+            $ad.css('height', closedHeight);
+            $('.ad-exp-collapse__slide', $fabricExpandableVideo).css(
+                'height',
+                closedHeight
+            );
+            if (params.trackingPixel) {
+                addTrackingPixel(params.trackingPixel + params.cacheBuster);
+            }
+            if (params.researchPixel) {
+                addTrackingPixel(params.researchPixel + params.cacheBuster);
+            }
+            $fabricExpandableVideo.appendTo(adSlot);
+            if (params.viewabilityTracker) {
+                addViewabilityTracker(
+                    adSlot,
+                    params.id,
+                    params.viewabilityTracker
+                );
+            }
+            adSlot.classList.add('ad-slot--fabric');
+            if (
+                adSlot.parentNode &&
+                ((adSlot.parentNode: any): Element).classList.contains(
+                    'top-banner-ad-container'
+                )
+            ) {
+                ((adSlot.parentNode: any): Element).classList.add(
+                    'top-banner-ad-container--fabric'
+                );
+            }
+            return true;
+        });
+    };
+
+    return Object.freeze({
+        create,
+    });
+};
+
+export default FabricExpandableVideoV2;
```

Comment:
@katebee Yeah I have been avoiding changing the default exports in the first PR as not to clash with other peoples changes in files...